recipes/pysbatch/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pysbatch" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/luptior/pysbatch/archive/{{ version }}.zip
+  sha256: d07cce60a6072846d5783f42bd690644bccd2b739634bf850ba158ee9113d30e
+
+build:
+  number: 0
+  skip: True  # [py<35]
```

Comment:
Please remove this selector and add ``noarch: python``

After that in ``host`` and also in ``run`` requirements put something like
```
- python >=3.5
```